src/Vsix/Merq.Vsix/Merq.Vsix.csproj

```diff
@@ -56,96 +60,62 @@
-    <PackageReference Include="Microsoft.VSSDK.BuildTools" Version="15.9.3039">
-      <PrivateAssets>all</PrivateAssets>
-      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
-    </PackageReference>
-    <PackageReference Include="MSBuilder.ThisAssembly.Project" Version="0.3.4" PrivateAssets="all" />
+    <PackageReference Include="Merq" Version="1.3.0" PrivateAssets="all" IncludeInVSIX="true" />
+    <PackageReference Include="Merq.Core" Version="1.3.0" PrivateAssets="all" IncludeInVSIX="true" />
+    <PackageReference Include="Merq.VisualStudio" Version="1.3.0" PrivateAssets="all" IncludeInVSIX="true" />
+    <PackageReference Include="Microsoft.VisualStudioEng.MicroBuild.Core" Version="1.0.0" PrivateAssets="all" />
+    <PackageReference Include="ThisAssembly.Project" Version="1.0.9" PrivateAssets="all" />
     <PackageReference Include="Clarius.VisualStudio" Version="2.0.14" />
-    <PackageReference Include="Xamarin.VSSDK" Version="0.4.0-alpha.24" />
-    <PackageReference Include="Xamarin.VSSDK.BuildTools" Version="0.4.0-alpha.24" />
-    <PackageReference Include="Microsoft.VSSDK.BuildTools" Version="17.1.4054" />
-    <PackageReference Include="Microsoft.VisualStudio.SDK" Version="17.1.32210.191" />
+    <PackageReference Include="Microsoft.VSSDK.BuildTools" Version="17.2.2190" />
+    <PackageReference Include="Microsoft.VisualStudio.SDK" Version="17.2.32505.173" />
     <PackageReference Include="Microsoft.VisualStudio.SDK.EmbedInteropTypes" Version="15.0.36" ExcludeAssets="all" />
   </ItemGroup>
 
   <ItemGroup>
     <BindingRedirect Include="Merq" />
     <BindingRedirect Include="Merq.Core" />
+    <BindingRedirect Include="Merq.VisualStudio" />
   </ItemGroup>
 
-  <!--
-    See https://github.com/xamarin/UITools/pull/705/commits/4a65bea18148f7ac2ecd34419450eda3550a776a by Thomas Sparks
-
-    TL;DR: ValidateVsixReferencedAssemblies locks the file, preventing Signing from succeeding
-  -->
-  <Target Name="TempRemoveVsixSourceItems" BeforeTargets="ValidateVsixReferencedAssemblies">
-    <ItemGroup>
-      <VSIXSourceItem Remove="$(IntermediateOutputPath)Merq.Vsix.dll"/>
-    </ItemGroup>
-  </Target>
-
-  <Target Name="ReturnVsixSourceItems" AfterTargets="ValidateVsixReferencedAssemblies">
-    <ItemGroup>
-      <VSIXSourceItem Include="$(IntermediateOutputPath)Merq.Vsix.dll"/>
-    </ItemGroup>
-  </Target>
-
-  <Target Name="GetFilesToSign" BeforeTargets="SignVsixContentFiles" Condition="'$(Configuration)' == 'Release'">
+  <Target Name="GetFilesToSign" BeforeTargets="SignVsixContentFiles" DependsOnTargets="IncludeNuGetResolvedAssets" Condition="'$(Configuration)' == 'Release'">
     <ItemGroup>
       <FilesToSign Include="$(IntermediateOutputPath)Merq.Vsix.dll">
         <Authenticode>Microsoft400</Authenticode>
       </FilesToSign>
+      <FilesToSign Include="@(MerqFile->'$(OutDir)%(Filename)%(Extension)')">
+        <Authenticode>Microsoft400</Authenticode>
```

Comment:
Authenticode should be `3PartySHA2`

Reply:
Done. How about the VsixSHA2? Is it still correct?

https://github.com/kzu/Merq/blob/dev/clean/src/Vsix/Merq.Vsix/Merq.Vsix.csproj#L94

Reply:
Yep, that one is fine